crates/matrix-sdk-ui/src/sync_service.rs

```diff
@@ -412,8 +412,8 @@ struct TerminationReport {
 #[doc(hidden)]
 impl SyncService {
     /// Return the existential states of internal tasks.
-    pub async fn task_state(&self) -> bool {
-        self.inner.lock().await.scheduler_task.is_some()
+    pub async fn is_supervisor_running(&self) -> bool {
```

Comment:
The doc comment needs an update here.

Reply:
5bd85fc9dac8f749630fff878349589c0c9a79cd